### PR TITLE
clarify how to enable VNC

### PR DESCRIPTION
**Pull request by ptmkenny**
This may be a rookie mistake, but I had trouble enabling VNC because I uncommented the "5900:5900" line and missed uncommenting "ports:".  Another reason I missed this is that the indentation is wrong; "ports:" has to be at the level of "external_links".  So I edited the comment to make it more clear about how to enable port 5900.

### CODE REVIEW

**Review by rfay — APPROVED** [review]
I don't see anything wrong with this change, but could you explain why you would use this instead of browser on `https://[DDEV SITE URL]:7900`? If there is significant value, please also add to the README explaining this approach. Note that when you start using `ports` only one project at a time can be running with port 5900.

**Comment by ptmkenny** [comment]
@rfay Ok, I updated the README with some basic guidelines that I hope are pretty easy to follow and non-controversial.

**Review by rfay — APPROVED** [review]
Great, thanks!

**Comment by tyler36** [comment]
Are there any additional steps needed?

I can visit `https://[project_name]:7900/` and login with secret. I can run my funcationJavascript tests, which pass/fail as expected but the `https://[project_name]:7900/` page only shows the "Selenium Grid" logo and title. 

selenium-chrome

**Comment by ptmkenny** [comment]
@tyler36 There shouldn't be any additional setup required. Make sure you aren't running your tests headless.  The tests that are shown are those that actually require the Chromium browser to be opened to run the tests, so if the tests are running headless in the background, they won't be shown.

**Comment by tyler36** [comment]
@ptmkenny Thanks for the suggestion.

By removing `\"--headless\"` from `.ddev/config.selenium-standalone-chrome.yaml`, and restarting DDEV to apply the updated config, I was able to watch the tests.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,32 @@ This service can be used with any project type. The examples below are Drupal-sp
...
     - `ddev exec -d /var/www/html/web "../vendor/bin/phpunit --bootstrap=../vendor/weitzman/drupal-test-traits/src/bootstrap-fast.php --printer '\Drupal\Tests\Listeners\HtmlOutputPrinter' ../vendor/weitzman/drupal-test-traits/tests/ExampleSelenium2DriverTest.php"`
-- On your host, browse to https://[DDEV SITE URL]:7900 (password: `secret`) to watch tests run (neat!).
+
+## Watching the tests
+
+### The easy way: Use noVNC (built-in)
+
+On your host, browse to https://[DDEV SITE URL]:7900 (password: `secret`) to watch tests run with noVNC (neat!).
+
+This is a no-configuration solution that enables you to quickly see what is going on with your tests.
```
Comment (tyler36) [inline README.md:34]: Considering we need to make configuration changes and restart DDEV, I'm not so sure about the wording here. 🤷

Comment (ptmkenny) [reply README.md:34]: Yeah, I didn't realize that `--headless` is set in the YAML file, so there should probably be a section on how and why to disable headless as well.  What's the reason for setting `--headless` as a default in the first place?  Is it just speed?

Comment (tyler36) [reply README.md:34]: I imaging there would be memory savings from using `--headless`, which might be more important when running on CI.

Note there are 2 references, one for Mink tests and one forr DTT tests.

@weitzman , are there issues removing this as the default?

**Comment by rfay** [comment]
Ping @weitzman before this gets too stale.

**Pull request merged**